com.unity.probuilder/Editor/EditorCore/EditorSceneViewPicker.cs

```diff
@@ -533,6 +548,34 @@ static float EdgeRaycast(Vector3 mousePosition, ScenePickerPreferences pickerPre
             return Mathf.Infinity;
         }
 
+        static Edge GetClosestEdgeToCamera(Vector3[] positions, IEnumerable<Edge> edges)
+        {
+            var camPos = SceneView.lastActiveSceneView.camera.transform.position;
+            var closestDistToScreen = Mathf.Infinity;
+            Edge closest = default;
```

Comment:
We still need to support legacy runtime for 2018.3, use `Edge.Empty`.

Reply:
Fixed d41affb